resources/TrialApp.didact.md

```diff
@@ -8,17 +8,17 @@ This tutorial will show you how to make and deploy changes to your Azure App Ser
 
 1. Locate your imported trial app in the Azure view.
 
-<img src="./trialAppTutorial/viewTrialAppScreenshot.png" style="width: 256px;">
+<img src="./trialAppTutorial/viewTrialAppScreenshot.png" style="width: 384px;">
```

Comment:
Usually we don't specify the width. Do you have to?

For example:
```
![Deploy button](resources/deploy-button.png)
```
https://raw.githubusercontent.com/microsoft/vscode-azureappservice/master/README.md

Reply:
To make the pictures larger and a consistent size I think I have to. If I don't then some are very very large and others are smaller etc.

Reply:
Ah - yeah I know what you're talking about. I think the trick is to change the size of the picture itself instead of specifying the style in the markdown. No need to ship a "very very large" picture in the vsix. Not sure how to do it on Windows, but on a Mac there's an "Adjust size" tool that makes it pretty easy

Reply:
Hmm. Doing that on Windows and it seems to reduce the quality substantially. I think I'd rather let the HTML + CSS rendering do the resize and have the image look nice. But maybe I'm not resizing correctly.

![image](https://user-images.githubusercontent.com/12476526/86663192-5206b580-bfa2-11ea-8950-d48f16927894.png)

Reply:
Yowza. Can you point me to the original image you just used and the exact pixel sizes you did? I can try

Reply:
https://github.com/microsoft/vscode-azureappservice/blob/master/resources/trialAppTutorial/viewTrialAppScreenshot.png

I resized the width to be 256 px, while maintaining the aspect ratio. This is the same resize that I have in the tutorial.